ql/src/java/org/apache/hadoop/hive/ql/exec/MemoryInfo.java

```diff
@@ -46,30 +48,25 @@ public MemoryInfo(Configuration conf) {
       llapInfo.initClusterInfo();
       if (llapInfo.hasClusterInfo()) {
         this.maxExecutorMemory = llapInfo.getMemoryPerExecutor();
+        LOG.info("Using LLAP registry executor MB {}", maxExecutorMemory / (1024L * 1024L));
```

Comment:
This just does not look correct to me.

Are you trying to log the value here in MB?  I think `this.maxExecutorMemory` is in MB already.  It is currently TB to MB with this large divisor.

Reply:
maxExecutorMemory should be in Bytes